Formula/libxlsxwriter.rb

```diff
@@ -0,0 +1,35 @@
+class Libxlsxwriter < Formula
+  desc "Libxlsxwriter: A C library for creating Excel XLSX files."
```

Comment:
Is "A C library for creating Excel .xlsx files" okay? That is the phrase I use in the docs.